cli/src/cli.rs

```diff
@@ -1144,6 +1146,13 @@ fn process_deploy(
         CliError::DynamicProgramError(format!("Unable to read program file: {}", err))
     })?;
 
+    let exe = EbpfVm::create_executable_from_elf(
+        &program_data,
+        Some(|x| bpf_verifier::check(x, true)));
+    exe.map_err(|err| {
+        CliError::DynamicProgramError(format!("ELF error: {}", err))
+    })?;
+
```

Comment:
I think we could just do this:
```suggestion
    let _ = EbpfVm::create_executable_from_elf(
        &program_data,
        Some(|x| bpf_verifier::check(x, true))).map_err(|err| {
        CliError::DynamicProgramError(format!("ELF error: {}", err))
    })?;

```